### PR TITLE
multiple blog posts converted to learn pages

### PR DESCRIPTION
**Pull request by serverless-mom**
I did not run the code snippets in each page, but they were working this year and should all be good. Framing on a few of the pages was shifted slightly. No edits were made to Stefan's posts.

## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [ ] Docs
* [x] Learn
* [ ] Other

### CODE REVIEW

File: site/content/learn/playwright/steps-decorators.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+
+But what if you want to give your test steps a human-readable name?
+
+### How to pass custom step names to a decorator
+
+To pass a custom step name to your decorator, you must change how you decorate the class methods. Instead of "just applying" a decorator with `@step` you can also execute your decorators with `@step()`.
+
+This change allows you to hand in arguments like a possible step name.
+
+```jsx
```
Comment (stefanjudis) [inline site/content/learn/playwright/steps-decorators.md:291]: Just curious why `jsx`? This should be `ts` or `typescript`? 🤔

**Comment by serverless-mom** [comment]
Oooh this was not intentional, I saw that Notion added markdown block
languages automagically, didn’t notice that it wasn’t quite right. Seems to
be rendering fine but I should fix that.

File: site/content/learn/playwright/test-fixtures.md
Comment (Dgiordano33) [inline site/content/learn/playwright/test-fixtures.md:1]: Could we remove all the puppeteer stuff here?

**Comment by serverless-mom** [comment]
pulled a small drive-by, following Tim's notes on a few docs I've looked at recently:

* In JS you almost always use `const` , unless you reassign a variable later. This rarely happens. So just use `const` always
* You can just use `ts` or `js` as the language in fenced code blocks, not `jsx`
* To keep code examples as clean as possible, we don't use the `;` at the end. With ES6 it's rarely needed.

**Comment by tnolet** [comment]
@serverless-mom you don't need to edit the existing page for syntax. I already did that. But please do it for new pages

**Pull request merged**